README.md

```diff
@@ -133,3 +133,23 @@ to restart Synapse to install the plugin.
 ## Development
 
 TODO. It's a TypeScript project with a linter.
+
+### Development and testing with mx-tester
+
+WARNING: mx-tester is currently work in progress, but it can still save you some time and is better than struggling with nothing.
+
+If you have docker installed you can quickly get setup with a development environment by using
+[mx-tester](https://github.com/matrix-org/mx-tester).
+
+Once you have mx-tester installed you we will need to build a synapse image with synapse_antispam.
+To do that you can just run `mx-tester build` in the mjolnir project root.
+Then we can start a container that uses that image and the config in `mx-tester.yml` in the project root by running `mx-tester up`.
+Then we can either run the integration tests `yarn test:integration` or inspect mojlnir ourselves by pointing a matrix client to http://localhost:9999 after running `yarn test:manual`.
+Once we are finished we can just run `mx-tester down`.
+
+### Running integration tests
+
+The integration tests can be run with `yarn test:integration`.
```

Comment:
Once you have called `mx-tester up`, you may run the integration tests with
...